yarn.lock

```diff
@@ -703,6 +703,13 @@
   "resolved" "https://registry.npmjs.org/axe-core/-/axe-core-4.3.5.tgz"
   "version" "4.3.5"
 
+"axios@^0.24.0":
```

Comment:
Let's avoid using axios and use fetch directly.

Reply:
Ok on it.